common/utils.js

```diff
@@ -2350,6 +2356,8 @@
             SESSION_VALIDATE: "session" + clientPathActionSeparator + "validate",
             SESSION_RETRIEVE: "session" + clientPathActionSeparator + "retrieve",
 
+            SWITCH_USER_ACCOUNTS_LOGIN: "switch-accounts" + clientPathActionSeparator + "login",
```

Comment:
I don't think this is used.

Reply:
I forgot to remove the `"action"` string places I added. Also after changing this variable name, I didn't go back and update it in the code.

Made the change and pushed it.

Reply:
Which means I need to update the logging document with the 2 new actions..

Reply:
Oh right. You just need to update the google sheet and add a new row after this one: